### PR TITLE
Add application working memo

### PR DESCRIPTION
**Pull request by matti-lamppu**
## 🛠️ Changelog
- Adds working memo to application

## 🗒️ Other notes
Additional notes if needed

## 🚚 Deployment reminder
- [ ] PR requires deployment changes
- [ ] Pipeline configuration change PR [link here]
- [ ] Azure devops library updated if needed
- [ ] Pipeline configuration change PR merged

## 🎫 Tickets
*This pull request resolves all or part of the following ticket(s):*
- [TILA-2898](https://helsinkisolutionoffice.atlassian.net/browse/TILA-2898)

[TILA-2898]: https://helsinkisolutionoffice.atlassian.net/browse/TILA-2898?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: tests/test_graphql_api/test_applications/test_update_permissions.py [hunk 1]
```diff
@@ -119,3 +119,89 @@ def test_service_sector_admin_for_other_sector_cannot_update_other_users_applica
...
+        "workingMemo": "123",
+    }
+    response = graphql(UPDATE_MUTATION, input_data=input_data)
+
+    # then:
+    # - The response contains errors about accessing the working memo
+    assert response.field_error_messages("workingMemo") == ["No permission to access working memo."]
+
+
+def test_general_admin_cannot_update_other_users_working_memo(graphql):
```
Comment (vergama) [inline tests/test_graphql_api/test_applications/test_update_permissions.py:186]: Typo: `cannot` => `can`

File: tests/test_graphql_api/test_applications/test_update_permissions.py [hunk 2]
```diff
@@ -119,3 +119,89 @@ def test_service_sector_admin_for_other_sector_cannot_update_other_users_applica
...
+        "workingMemo": "123",
+    }
+    response = graphql(UPDATE_MUTATION, input_data=input_data)
+
+    # then:
+    # - The response contains no errors
+    assert response.has_errors is False, response
+
+
+# TODO: Permissions to update working memo
```
Comment (vergama) [inline tests/test_graphql_api/test_applications/test_update_permissions.py:207]: Something missing or comment left hanging?

File: tests/test_graphql_api/test_applications/test_update_permissions.py (hunk 1 above)
Comment (matti-lamppu) [reply tests/test_graphql_api/test_applications/test_update_permissions.py]: Fixed

File: tests/test_graphql_api/test_applications/test_update_permissions.py (hunk 2 above)
Comment (matti-lamppu) [reply tests/test_graphql_api/test_applications/test_update_permissions.py]: Left hanging, tests were written above. Removed this.

**Review by vergama — APPROVED** [review]

**Comment by vergama** [comment]
Tiedon `application.working_memo` voisi tuoda hakemuksesta automaattisesti (hakukierroksen käsittelyn päätytyyä) luotujen varausten tiedon `reservation.working_memo` arvoksi. (ApplicationEventScheduleActions:create_reservation_for_schedule)

**Review by vergama — APPROVED** [review]

**Pull request merged**